docs/clusterfuzzlite/build_integration.md

```diff
@@ -33,10 +40,17 @@ Before you can start setting up your new project for fuzzing, you must do the fo
 - Clone the OSS-Fuzz repo: `git clone https://github.com/google/oss-fuzz.git`
 
 ## Generating an empty build integration
-Build integrations consist of two files, a `Dockerfile` that can install
-dependencies for your project, and a `build.sh` file that actually builds your
-project. These must live in the `.clusterfuzzlite` directory in the root of your
+
+Build integrations consist of three configuration files:
+* [./clusterfuzzlite/project.yaml](#projectyaml) - provides metadata about the project.
```

Comment:
Fine for now I guess.

Reply:
Another reason to keep project.yaml: In the future we may want auto-bugfiling and specifying CCs (or github usernames). There's not really much of a cost apart there being an auto-generated file in most cases is there?